openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion.py

```diff
@@ -108,6 +111,32 @@ def student_view(self, context=None):
         """
         fragment = Fragment()
 
+        # Head dependencies
+
+        discussion_vendor_js = ['js/jquery.autocomplete.js', 'js/src/tooltip_manager.js']
+
+        discussion_vendor_js += settings.PIPELINE_JS['discussion_vendor']['source_filenames']
+
+        for vendor_js_file in discussion_vendor_js:
+            fragment.add_resource_url(static(vendor_js_file), "application/javascript", "head")
+
+        discussion_vendor_css = 'css/vendor/jquery.autocomplete.css'
+        fragment.add_css_url(static(discussion_vendor_css))
+
+        # Body dependencies
+
+        discussion_js = settings.PIPELINE_JS['discussion']['source_filenames']
+
+        for js_file in discussion_js:
+            fragment.add_javascript_url(static(js_file))
+
+        fragment.add_javascript(loader.render_template("static/js/discussion_classes.js"))
```

Comment:
@itsjeyd `discussion_classes.js` is not a template - you might want to use `loader.load_unicode` (can't tell the exact name off top of my head, but this is likely accurate)